application/F3DOptionsParser.cxx

```diff
@@ -213,72 +215,72 @@ void ConfigurationOptions::GetOptionsFromArgs(
-    this->DeclareOption(grp2, "texture-normal", "", "Path to a texture file that sets the normal map of the object", options.getAsStringRef("texture-normal"), HasDefault::NO, MayHaveConfig::YES, "<file path>");
-    this->DeclareOption(grp2, "normal-scale", "", "Normal scale affects the strength of the normal deviation from the normal texture", options.getAsDoubleRef("normal-scale"), HasDefault::YES, MayHaveConfig::YES, "<normalScale>");
+    this->DeclareOption(grp2, "point-sprites", "o", "Show sphere sprites instead of geometry", options.getAsBoolRef("window.point-sprites.enable"), HasDefault::YES, MayHaveConfig::YES);
+    this->DeclareOption(grp2, "point-size", "", "Point size when showing vertices or point sprites", options.getAsDoubleRef("loader.default.point-size"), HasDefault::YES, MayHaveConfig::YES, "<size>");
+    this->DeclareOption(grp2, "line-width", "", "Line width when showing edges", options.getAsDoubleRef("loader.default.line-width"), HasDefault::YES, MayHaveConfig::YES, "<width>");
+    this->DeclareOption(grp2, "color", "", "Solid color", options.getAsDoubleVectorRef("loader.default.color.rgb"), HasDefault::YES, MayHaveConfig::YES, "<R,G,B>");
+    this->DeclareOption(grp2, "opacity", "", "Opacity", options.getAsDoubleRef("loader.default.color.opacity"), HasDefault::YES, MayHaveConfig::YES, "<opacity>");
+    this->DeclareOption(grp2, "roughness", "", "Roughness coefficient (0.0-1.0)", options.getAsDoubleRef("loader.default.material.roughness"), HasDefault::YES, MayHaveConfig::YES, "<roughness>");
+    this->DeclareOption(grp2, "metallic", "", "Metallic coefficient (0.0-1.0)", options.getAsDoubleRef("loader.default.material.metallic"), HasDefault::YES, MayHaveConfig::YES, "<metallic>");
+    this->DeclareOption(grp2, "hdri", "", "Path to an image file that will be used as a light source", options.getAsStringRef("window.background.hdri"), HasDefault::NO, MayHaveConfig::YES, "<file path>");
+    this->DeclareOption(grp2, "texture-base-color", "", "Path to a texture file that sets the color of the object", options.getAsStringRef("loader.default.color.texture"), HasDefault::NO, MayHaveConfig::YES, "<file path>");
+    this->DeclareOption(grp2, "texture-material", "", "Path to a texture file that sets the Occlusion, Roughness and Metallic values of the object", options.getAsStringRef("loader.default.material.texture"), HasDefault::NO, MayHaveConfig::YES, "<file path>");
+    this->DeclareOption(grp2, "texture-emissive", "", "Path to a texture file that sets the emitted light of the object", options.getAsStringRef("loader.default.emissive.texture"), HasDefault::NO, MayHaveConfig::YES, "<file path>");
+    this->DeclareOption(grp2, "emissive-factor", "", "Emissive factor. This value is multiplied with the emissive color when an emissive texture is present", options.getAsDoubleVectorRef("loader.default.emissive.factor"), HasDefault::YES, MayHaveConfig::YES, "<R,G,B>");
```

Comment:
`loader.material.emissive` ?

Reply:
In any case, we should unify. This emissive factor should act just like other uniform coefficient like color/opacity/roughness/metallic

Reply:
this seems to be already the case